changelog/pending/20230911--cli-config-display--adds-display-of-query-individual-org-validation-console-query-syntax-alignment.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: cli/config,display
+  description: Adds display of query, individual org validation, console query syntax alignment
```

Comment:
Do we need this changelog entry?

Further, should we cleanup some of the other changelog files? Right now, the draft of the release show's a whole bunch of random things:

<img width="589" alt="Screen Shot 2023-09-11 at 11 02 56 AM" src="https://github.com/pulumi/pulumi/assets/710598/6ac9f6ba-3046-4c52-a690-65173c621949">

@AaronFriel, how would you like the changelog to look for this release w.r.t to these features?